meta-ros1-melodic/recipes-bbappends/mavros/mavros_0.33.3-1.bbappend

```diff
@@ -0,0 +1,5 @@
+DEPENDS_remove = "${ROS_UNRESOLVED_PLATFORM_PKG_geographiclib-tools}"
+
+ROS_EXEC_DEPENDS = " \
```

Comment:
Here you're overwriting all the 20 runtime dependencies from  mavros_0.33.3-1.bb, that's probably not what you wanted.

Hint: always use "bitbake -e" to check that it does what you think it should.